www/content/install/macos_x86_64.md

```diff
@@ -18,12 +18,6 @@ which includes the Roc compiler and some helpful utilities.
     cd roc_night<TAB TO AUTOCOMPLETE>
     ```
 
-1. Install required dependencies:
-
-   ```sh
-   brew install z3 zstd
```

Comment:
Don't we need z3 anymore either?

Reply:
Seems like no, but I removed all usages and installs of `z3` I saw. If it works, we don't need it either.